src/powfacpy/results_interface.py

```diff
@@ -70,7 +70,10 @@ def _get_time_variable_name_from_elmres(elmres):
       29:'b:ucttime' # quaidynamic simulation
       # TODO to be continued if needed
     }
-    return time_names[simulation_type_number]
+    try:
+      return time_names[simulation_type_number]
+    except KeyError:
+      print('PF simulation type number {} not known or not implemented yet. Maybe to be added to time_names dictionary?')
```

Comment:
This will not throw an exception if a KeyError is raised. 
Should it be an f-string with "{elmres.calTp}"?

Reply:
I forgot to include, I'll do a fix. 
Do you think it's fine like this? 

```
    try:
      return time_names[simulation_type_number]
    except KeyError:
      print('PF simulation type number {} not known or not implemented yet. Maybe to be added to time_names dictionary?'.format(simulation_type_number))
```